.pre-commit-config.yaml

```diff
@@ -287,51 +287,5 @@ repos:
     - --cobertura-xml-report=.tox/.tmp/.test-results/mypy--py-3.9
     - --html-report=.tox/.tmp/.test-results/mypy--py-3.9
     pass_filenames: false
-- repo: https://github.com/pre-commit/mirrors-mypy.git
-  # NOTE: This is kept at v1.5.1 until we stop supporting Python 3.6.
-  rev: v1.5.1  # enforce-version: v1.5.1
```

Comment:
Let's also drop `id: enforced-versions` at the beginning of the config, now that it's unused.